org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/capabilities/codelens/ClientCommands.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package org.eclipse.lsp4xml.settings.capabilities.codelens;

```

Comment:
I'd put it either in `org.eclipse.lsp4xml` or `org.eclipse.lsp4xml.client`, since we might add other commands not specific to codelenses

Reply:
@fbricon updated in `org.eclipse.lsp4xml.client`